bot.go

```diff
@@ -28,6 +33,7 @@ func NewBot(pref Settings) (*Bot, error) {
 		handlers: make(map[string]interface{}),
 		stop:     make(chan struct{}),
 		reporter: pref.Reporter,
+		client:client,
```

Comment:
This should be indented properly.
(You could also fix the indentation of Token, Updates and Poller by adding an extra space)